contracts/test/MockSpokePool.sol

```diff
@@ -19,4 +21,13 @@ contract MockSpokePool is SpokePoolInterface, SpokePool {
     function _bridgeTokensToHubPool(RelayerRefundLeaf memory relayerRefundLeaf) internal override {}
 
     function _requireAdminSender() internal override {}
+
+    function chainId() public view override(SpokePool, SpokePoolInterface) returns (uint256) {
```

Comment:
nit you can remove the override of the `SpokePoolInterface` if you remove this from the contract. I don't think the other SpokePools extend `SpokePoolInterface` anyways